### PR TITLE
Theme Cicada/Vicuna little design update

### PR DESCRIPTION
**Pull request by opnsenseuser**
Theme Cicada/Vicuna little design update

included requested cross-out/strikeout text in the row for disabled rules/nat
https://forum.opnsense.org/index.php?topic=21634.15

### CODE REVIEW

**Comment by opnsenseuser** [comment]
@fichtner I'm a little disappointed that you haven't merged the two theme updates for the next version.

File: misc/theme-vicuna/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-vicuna
-PLUGIN_VERSION=		1.3
+PLUGIN_VERSION=		1.31
```
Comment (fichtner) [inline misc/theme-vicuna/Makefile:2]:
```suggestion
PLUGIN_VERSION=		1.4
```

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by opnsenseuser** [comment]
sorry, something did go wrong with the merging!

```
# opnsense-patch -c plugins c8b20efd
Found local copy of c8b20efd, skipping fetch.
No such line 10646 in input file, ignoring
51 out of 278 hunks failed while patching opnsense/www/themes/cicada/assets/styl                                              esheets/main.scss
19 out of 21 hunks failed while patching opnsense/www/themes/cicada/build/css/ma                                              in.css
```

**Comment by opnsenseuser** [comment]
@fichtner do you need the branch again to get rid of the merging problem? Do i need to upload the code again? See the errors above. regards rené

**Comment by AdSchellevis** [comment]
@opnsenseuser best make sure the master branch is in the state you would like to see it in before the next release slot. you can always build a theme package manually on your end. A proper use of `opnsense-patch` comes with strict management from the contributors end.

**Comment by opnsenseuser** [comment]
@AdSchellevis should i upload both themes again on github?

**Pull request merged**